ftw/news/browser/resources/news.scss

```diff
@@ -2,69 +2,89 @@
 @include portal-type-font-awesome-icon(ftw-news-newsfolder, newspaper-o);
 @include portal-type-font-awesome-icon(ftw-news-newslistingblock, newspaper-o);
 
-.portletItemDate {
-  color: $color-text-light;
-  font-size: $font-size-small;
-  margin-bottom: $margin-heading-vertical;
-}
+$leadimage-width: 200px;
```

Comment:
By adding `!default` we could make this variable customizable. At the moment it is not customizable.

I'd also `declare-variables` :wink: